src/test/miniscript_tests.cpp

```diff
@@ -120,6 +120,8 @@ struct KeyConverter {
 //! Singleton instance of KeyConverter.
 const KeyConverter CONVERTER{};
 
+// https://github.com/llvm/llvm-project/issues/53444
+// NOLINTNEXTLINE(misc-unused-using-decls)
```

Comment:
The issue necessitating the NOLINTNEXTLINE added here is fixed in llvm 16. See https://github.com/llvm/llvm-project/issues/55095.

Reply:
Followed up with in #27404.